pkgs/development/python-modules/django-maintenance-mode/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage,
+  mock, django }:
+buildPythonPackage rec {
+  pname = "django-maintenance-mode";
+  version = "0.13.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16yfw85r6my14vbrf9g2hj5p7ivgjvrd16s9prsjzqj9k8qikch2";
+  };
+
+  doCheck = false;
```

Comment:
The tests just aren't included in the pypi package - they run fine if you fetch the source from github instead:

```
  src = fetchFromGitHub {
    owner = "fabiocaccamo";
    repo = pname;
    rev = version;
    sha256 = "06li0rbfc8f68y472dvp95sb5c9s5b15a4zjb2xdi5a7fndra4lr";
  };
```

Also note that the latest available version is 0.13.3

Reply:
Nice, I'll add them, thank you :pray: